### PR TITLE
STCOM-948: Add 'aria-expanded' attr to IconButton

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
## Purpose
There is an accessibility issue in users app Open Loans List: "Elements must only use allowed ARIA attributes". This happens because the button's 'aria-expanded' attribute is extended to child elements.

## Approach
- Catch the attribute and assign it manually to the button

## Ref
https://issues.folio.org/browse/STCOM-948

### CODE REVIEW

**Review by zburke — CHANGES_REQUESTED** [review]
Thank you for a clear explanation of the problem and how this change addresses it! Pulling `aria-expanded` out of the values that cascade down makes sense to me. @JohnC-80 , do you agree? 

Please add a prop-types entry for this to clean up the lint warnings.

File: lib/IconButton/IconButton.js [hunk 1]
```diff
@@ -86,6 +88,7 @@ const IconButton = React.forwardRef(({
 IconButton.propTypes = {
   'aria-label': PropTypes.string,
   'aria-labelledby': PropTypes.string,
+  'aria-expanded': PropTypes.string,
```
Comment (zburke) [inline lib/IconButton/IconButton.js:91]: Pedantic: as noted by lint prop-types need to be sorted alphabetically so this should be added at the top of the list.

**Review by zburke — CHANGES_REQUESTED** [review]
So close! Please clean up the prop-types ordering and then I think we're done. Agreed, @JohnC-80?

**Review by JohnC-80 — APPROVED** [review]
#1737 was a similar issue - it adds `aria-haspopup` as well.
If we get many more of these, we might do something to consolidate the `aria-` props somehow without adding new proptypes, but for now, this is the way....

**Comment by mariia-aloshyna** [comment]
Thanks for review @zburke and @JohnC-80! Prop-types are cleaned up.

**Pull request merged**